docs/react-jss.md

```diff
@@ -117,15 +116,15 @@ and
 
 #### Dynamic values
 
-You can use [function values](jss-syntax.md#function-values), function rules and observables out of the box. Function values and function rules will receive a props object once the component receives new props or mounts for the first time.
+You can use [function values](jss-syntax.md#function-values), Function rules and observables out of the box. Function values and function rules will receive a props object once the component receives new props or mounts for the first time.
```

Comment:
is it a correct way to write it? ", Function rules and observables out of the box." I don't know just asking because we have inconsistency across thee docs and probably should make it consistent.

Reply:
Sometimes I think it should have been called rule functions and value functions